### ISSUE TITLE
Speed up the querying process

### ISSUE THREAD

**Issue by bsrinivasa** [amishas157/lookup-osm-wikidata#4]
@amishas157 ,
Would it be possible to reduce the query and fetch time by running parallel processes as discussed earlier? Currently osm.py can process ~44 features (OSM nodes) in a minute.

As the number of features have been increasing in the newer lists, this would be beneficial to lookup huge numbers quickly.

**Comment by amishas157** [comment]
@bsrinivasa 
Yes, this looks [possible](https://docs.python.org/3/library/multiprocessing.html) . Need bandwidth to learn and implement it. Will keep this ticket updated. Thanks for the heads up.
